stan/math/prim/prob/inv_wishart_cholesky_lpdf.hpp

```diff
@@ -0,0 +1,105 @@
+#ifndef STAN_MATH_PRIM_PROB_INV_WISHART_CHOLESKY_LPDF_HPP
+#define STAN_MATH_PRIM_PROB_INV_WISHART_CHOLESKY_LPDF_HPP
+
+#include <stan/math/prim/meta.hpp>
+#include <stan/math/prim/err.hpp>
+#include <stan/math/prim/fun/dot_self.hpp>
+#include <stan/math/prim/fun/dot_product.hpp>
+#include <stan/math/prim/fun/lmgamma.hpp>
+#include <stan/math/prim/fun/mdivide_left_tri.hpp>
+#include <stan/math/prim/fun/constants.hpp>
+
+namespace stan {
+namespace math {
+
+/** \ingroup multivar_dists
+ * Return the natural logarithm of the unnormalized inverse wishart density of
+ the specified
+ * lower-triangular Cholesky factor variate, positive degrees of freedom, and
+ lower-triangular
+ * Cholesky factor of the scale matrix.
+ *
+ * The scale matrix, `L_S`, must be a lower Cholesky factor.
+ * Dimension, k, is implicit.
+ * nu must be greater than k-1
+ *
+ * The change of variables from Y, a positive-definite matrix, to
+ * L_Y, the lower triangular Cholesky factor, is given in Theorem 2.1.9 in
+ * Muirhead, R. J. (2005).
+ * Aspects of Multivariate Statistical Theory. Wiley-Interscience.
+
+ * @tparam T_y Cholesky factor matrix
+ * @tparam T_dof scalar degrees of freedom
+ * @tparam T_scale Cholesky factor matrix
+ * @param L_Y lower triangular Cholesky factor of a covariance matrix
+ * @param nu scalar degrees of freedom
+ * @param L_S lower triangular Choleskyy factor of the scale matrix
+ * @return natural logarithm of the Wishart density at L_Y given nu and L_S
+ * @throw std::domain_error if nu is not greater than k-1
+ * @throw std::domain_error if L_S is not a valid Cholesky factor.
+ */
+template <bool propto, typename T_y, typename T_dof, typename T_scale,
+          require_stan_scalar_t<T_dof>* = nullptr,
+          require_all_matrix_t<T_y, T_scale>* = nullptr>
+return_type_t<T_y, T_dof, T_scale> inv_wishart_cholesky_lpdf(
+    const T_y& L_Y, const T_dof& nu, const T_scale& L_S) {
+  using Eigen::Lower;
+  using T_L_Y_ref = ref_type_t<T_y>;
+  using T_nu_ref = ref_type_t<T_dof>;
+  using T_L_S_ref = ref_type_t<T_scale>;
+  using T_return = return_type_t<T_y, T_dof, T_scale>;
+  static const char* function = "inv_wishart_cholesky_lpdf";
+  Eigen::Index k = L_Y.rows();
+  check_size_match(function, "Rows of random variable", L_Y.rows(),
+                   "columns of scale parameter", L_S.rows());
+  check_size_match(function, "Rows of random variable", L_Y.rows(),
+                   "columns of random variable", L_Y.cols());
+  check_size_match(function, "Rows of random variable", L_S.rows(),
+                   "columns of random variable", L_S.cols());
+
+  T_L_Y_ref L_Y_ref = L_Y;
+  T_nu_ref nu_ref = nu;
+  T_L_S_ref L_S_ref = L_S;
+
+  check_greater(function, "Degrees of freedom parameter", nu_ref, k - 1);
+  check_positive(function, "Cholesky Random variable", L_Y_ref.diagonal());
+  check_positive(function, "columns of Cholesky Random variable",
```

Comment:
Random is sometimes capitalized but not always in these check calls. Let's make this consistent. It seems to me that we use the capitalized version in other checks across math?

Reply:
so I shouldn't change anything?

Reply:
It's not capitalized in lines 53-58.